extension.yaml

```diff
@@ -192,32 +195,240 @@ params:
     default: subscribed
     required: true
 
-  - param: FIREBASE_MEMBER_TAGS_CONFIG
+  - param: MAILCHIMP_MEMBER_TAGS_WATCH_PATH
+    label: Firebase Member Tags Watch Path
+    description: >-
+      Specify the Firestore collection to listen for member tag changes
+
+
+      Firebase functions must listen to a document path. As such, a specific id (registrations/CRI1YQ0S7YkZhuGkJCDN) or wildcard path (registrations/{documentId}) must be specified
+      
+      
+      learn more: https://firebase.google.com/docs/firestore/extend-with-functions#wildcards-parameters
+
+
+      NOTE: To opt-out of this feature by disabling the function listener, enter `N/A`.
+
+
+    type: string
+    example: registrations/{documentId}
+    default: N/A
```

Comment:
You should set a default. Definitely don't leave it as "N/A" or _that_ will be the default. :)

Reply:
Oh "N/A" is a valid value. Never mind!